Helper/Cart.php

```diff
@@ -321,27 +368,41 @@ public function getCartData($payment_only, $place_order_payload)
 
             $product['reference']    = $productId;
             $product['name']         = $item->getName();
-            $product['description']  = $item->getDescription();
             $product['total_amount'] = $rounded_total_amount;
             $product['unit_price']   = $this->getRoundAmount($unit_price);
             $product['quantity']     = round($item->getQty());
-            $product['sku']          = $item->getSku();
-            //Get product Image
-            $_product = $this->productModel->load($productId);
-            $product['image_url'] = $this->imageHelper->init($_product, 'category_page_list')->getUrl();
+            $product['sku']          = trim($item->getSku());
+
+            ////////////////////////////////////
+            // Get product description and image
+            ////////////////////////////////////
+            /**
+             * @var \Magento\Catalog\Model\Product
+             */
+            $_product = $this->productFactory->create()->load($productId);
+            // ignore description, reduce the payload until its use is confirmed
```

Comment:
It is used in Bolt backend. Please include.

Reply:
done